etc/policy.template.toml

```diff
@@ -61,19 +61,10 @@ required = false
 # - 'CASCADE_SERVER': The TCP/UDP port where Cascade is serving the zone for
 #     review, formatted '<ip-addr>:<port>'.
 #     TODO: Multiple such servers can be configured, which one should we pick?
-# - 'CASCADE_CONTROL': The address of Cascade's HTTP API server, for sending
-#     approvals and rejections.
 #
 # The command will be called from an unspecified directory, and it must be
-# accessible to Cascade (i.e. after it has dropped privileges).
-#
-# To approve the zone contents, send an HTTP GET request to
-# `http://$CASCADE_CONTROL/approve/$CASCADE_TOKEN?zone=$CASCADE_ZONE&serial=$CASCADE_SERIAL`.
-# `approve` can be replaced by `reject` to reject the zone.  If the command
-# exits without approving or rejecting the zone, Cascade will wait for the user
-# to approve it manually.
-#
-# TODO: Use the exit code of the hook to infer approval / rejection?
+# accessible to Cascade (i.e. after it has dropped privileges).  Its exit code
+# will determine whether the zone is approved or not.
 #cmd-hook = "review-unsigned-zone.sh"
```

Comment:
I'm not realizing that `cmd-hook` as a name is a bit weird. It's a hook or a command but a bit redundant to call it both, while also "cmd" is a bit vague. Maybe we should rename this to `review-script-path`?